sycl/doc/extensions/experimental/sycl_ext_oneapi_raw_kernel_arg.asciidoc

```diff
@@ -44,11 +44,17 @@ SYCL specification refer to that revision.
 
 == Status
 
-This is a proposed extension specification, intended to gather community
-feedback.  Interfaces defined in this specification may not be implemented yet
-or may be in a preliminary state.  The specification itself may also change in
-incompatible ways before it is finalized.  *Shipping software products should
-not rely on APIs defined in this specification.*
+This is an experimental extension specification, intended to provide early
+access to features and gather community feedback.  Interfaces defined in this
+specification are implemented in {dpcpp}, but they are not finalized and may
+change incompatibly in future versions of {dpcpp} without prior notice.
+*Shipping software products should not rely on APIs defined in this
+specification.*
+
+[NOTE]
+====
+This extension is currently only considered supported by the Level-Zero backend.
+====
```

Comment:
Rather than adding a note like this, we usually add a section "Backend support status" as shown [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc#backend-support-status) in the template.